<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_Height">
      <property name="text">
       <string>Height (cm)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label2_Interpretation">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSlider" name="slider_Weight">
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_Gender">
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QSpinBox" name="spinBox_Height">
      <property name="suffix">
       <string> cm</string>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QSlider" name="slider_Height">
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_BMI">
      <property name="text">
       <string>Body mass index</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label2_BMI">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_Interpretation">
      <property name="text">
       <string>Interpretation</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="label2_Gender">
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Weight">
      <property name="text">
       <string>Weight (kg)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QSpinBox" name="spinBox_Weight">
      <property name="suffix">
       <string> kg</string>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuGender">
    <property name="title">
     <string>Gender</string>
    </property>
    <addaction name="actionFemale"/>
    <addaction name="actionMale"/>
   </widget>
   <addaction name="menuAction"/>
   <addaction name="menuGender"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionFemale">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="dot.qrc">
     <normaloff>:/dot/images.png</normaloff>:/dot/images.png</iconset>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </action>
  <action name="actionMale">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="dot.qrc">
     <normaloff>:/dot/images.png</normaloff>:/dot/images.png</iconset>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dot.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>spinBox_Weight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_Weight</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Height</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_Height</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_Weight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Weight</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_Height</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Height</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
